progect-management-app/src/app/app.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  title = 'project-management-app';
```

Comment:
It better to use modifiers and typing for each property or method. It leads to better readability of code.
```public title: string = 'project-management-app'```